pkg/workloads/console/runner/runner.go

```diff
@@ -232,11 +228,36 @@ func (c *Runner) Create(ctx context.Context, opts CreateOptions) (*workloadsv1al
 				Hook:       opts.Hook,
 			},
 		)
+		if err != nil {
+			return csl, err
+		}
+
+		return csl, c.exitWithStatus(csl)
 	}
 
 	return csl, nil
 }
 
+func (c *Runner) exitWithStatus(csl *workloadsv1alpha1.Console) error {
```

Comment:
The naming of this thing feels odd, it doesn't do what I expect it might do from the name alone.

I'm not really sure of a better function name though - `errorIfUnsuccessfulCompletion`?

Reply:
I went for `waitForSuccess`